addons/crm/models/crm_lead.py

```diff
@@ -541,7 +541,7 @@ def _search_is_lost(self, operator, value):
     @api.depends('stage_id.is_won', 'probability')
     def _compute_is_won(self):
         for lead in self:
-            lead.is_won = lead.probability == 100 and lead.stage_id.is_won
+            lead.is_won = lead.stage_id.is_won and lead.probability == 100
```

Comment:
Strange change inside this commit ^^

Will not matter if we squash commits, but I think keeping probability first is better for performance, as if it's not 100%, the code will not need to fetch the stage_id at all.

Reply:
Hello @awa-odoo 

I'm done, just posting a reply to this very comment. Actually, the query counters went up in two tests linked to _action_assign_leads. (1176 > 1167 and 6303 > 6066) When trying to switch stage and proba, it did not, hence the change.

I guessed this had to do with the probability being a computed field, even though stored. As we have queries in the pls flow, _compute_probabilities could lead to extra queries in comparison to reading the stage. I can investigate more if needed.

What do you think? I can at least already move it to Testing BE?

Cheers!

Reply:
Fine then, just seemed strange in that commit context, but indeed computed field can add queries so switching the check order actually matters.

Greened on my side, thanks again! 🟢